src/main/java/bowling/domain/NormalFrame.java

```diff
@@ -0,0 +1,45 @@
+package bowling.domain;
+
+import bowling.domain.state.Ready;
+import bowling.domain.state.State;
+
+public class NormalFrame implements Frame {
+    private State state;
+    private Frame nextFrame;
+    private static final int FINAL_NORMAL_FRAME = 9;
+    private final int round;
```

Comment:
```suggestion
public class NormalFrame implements Frame {
    private static final int FINAL_NORMAL_FRAME = 9;

    private State state;
    private final int round;
```

NormalFrame 을 상태패턴의 Context 로 만들어주셨군요!
Context 에서 변화하는 것은 State 만 변화하도록 하고,
나머지는 final 로 혹은 멤버가 존재하지 않도록 하는 것은 어떨지 제안해봅니다.